Makefile

```diff
@@ -30,14 +30,19 @@ CI_FLAGS =\
 .PHONY: dev_req
 dev_req: ## Installs dbt-* packages in develop mode along with only development dependencies.
 	@\
-	pip install -r dev-requirements.txt
-	pip install -r editable-requirements.txt
+	pip install -e ./core -r dev-requirements.txt
```

Comment:
We need to install all of this in one go.
- `dbt-core` depends on `dbt-tests-adapter` for testing
- `dbt-tests-adapter` depends on `dbt-core`

It's worth noting that `tox` does this slightly differently, but it results in the same outcome. `make unit` uses `tox`. This is fixing the local virtual environment (separate but related).

Reply:
Should we use `editable-requirements.txt` here for consistency?

Reply:
I think that's what's in tox and the workflows, so that might make sense. However, `editable-requirements.txt` is now only one package since `dbt-tests-adapter` and `dbt-postgres` have be migrated out. Consistency makes sense either way.